src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -2971,7 +2971,7 @@ private void AddToJobRunningList(PSJobStartEventArgs jobArgs, DebuggerResumeActi
                     startAction,
                     _context.EngineHostInterface.ExternalHost,
                     _context.SessionState.Path.CurrentLocation,
-                    GetFunctionToSourceMap());
+                    new Dictionary<string, DebugSource>());
```

Comment:
It's not obvious from the diffs, but it looks like this dictionary might not be needed anymore.
If it's needed and clear elsewhere in the code, then fine, but if it's not needed or unclear, a comment would be good.

Reply:
Yes, I noticed that too. The implementation of `SetParent` in `RemotingJobDebugger` just throws away all the arguments. However, `SetParent` is a virtual method, theoretically, someone could derive the `Debugger` class and implement `SetParent` that actually use the dictionary ... So, it's a breaking change that I'm not sure if I can make.

@PaulHigin Is it OK to remove the `functionSourceMap` parameter from `public virtual void SetParent`?
If it's OK, I can clean up the `DebugSource` type as a whole.

Reply:
@PaulHigin Could you please take a look and weigh in? Thanks!

Reply:
Yes, the GetFunctionToSourceMap() is only for the workflow debugger.  You can remove the dictionary from SetParent() and also the GetFunctionSource() private method.

Reply:
Thanks @PaulHigin for the insight!
I have addressed your comment. Also removed the `DebugSource` type, which is not used anywhere else.